scenarios/ecs_privesc_evade_protection/terraform/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
```

Comment:
Add required providers for aws, time, and null (archive see other comment)

https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.4.0/docs/rules/terraform_required_providers.md